clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -37,8 +37,18 @@
      * @param conf          The configuration.
      * @return              The new KafkaAdminClient.
      */
+    public static AdminClient create(AdminClientConfig conf) {
```

Comment:
It seems like there's no way to create an instance of AdminClientConfig or am I missing something?